src/Lms/Canvas/CanvasLms.php

```diff
@@ -76,7 +76,8 @@ public function getId()
     public function getLtiAuthUrl($globalParams)
     {
         $session = $this->sessionService->getSession();
-        $baseUrl = $session->get('iss');
+        $baseUrl = $_ENV['JWK_BASE_URL'] ?: $session->get('iss');
```

Comment:
This will return a notice if `JWK_BASE_URL` is undefined in .env. Since the variable is optional let's try:
```
$baseUrl = !empty($_ENV['JWK_BASE_URL']) ? $_ENV['JWK_BASE_URL'] : $session->get('iss');
```
This will ignore `JWK_BASE_URL` if it's an empty string, null, or undefined.

Reply:
I agree with this suggestion; I'm also switching to `getenv()` as that appears to be the modern best practice.